tests/cases/compiler/contextualTypingReturnStatementWithReturnTypeAnnotation.ts

```diff
@@ -0,0 +1,18 @@
+// @strict: true
+// @noEmit: true
```

Comment:
```suggestion
// @emitDeclarationOnly: true
```

Reply:
Out of curiosity - any particular reason why you'd like to test the declaration emit here?

@jakebailey also raised [here](https://github.com/microsoft/TypeScript/pull/52628#issuecomment-1418723831) a good question if this should even type check. Do you have any thoughts on that? I have a hard time verifying this - it feels to me that perhaps it shouldn't as there are no strong guarantees that this doesn't introduce accidental runtime errors.

Reply:
Actually, never mind. I realized that declaration emit won't be directed by inference anyway